projects/Mallard/src/components/onboarding/onboarding-card.tsx

```diff
@@ -45,7 +45,7 @@ const appearances: {
         text: { color: color.palette.neutral[100] },
     }),
     [CardAppearance.blue]: StyleSheet.create({
-        background: { backgroundColor: '#5da8db' },
+        background: { backgroundColor: '#399fdc' },
```

Comment:
Harcoded colour?

Reply:
I can't find this colour anywhere else so thought I'd leave it here as I don't know what it represents - I'm not quite sure where to move it to otherwise as it doesn't have any sort of semantic meaning to hook it into in our generic palette.